### DISCUSSION TITLE
Custom email invitation content.

### DISCUSSION THREAD

**Discussion by vlaguduva** [grafana/grafana#38360] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
I would like to customize the email content, to be precise I would like to have the ingress/nginx url as .LinkUrl instead of localhost:3000.

**Why is this needed**:
When grafana used behind Nginx/Ingress controller in kubernetes, we often do not touch the contents of grafana.ini which means the domain, port stays as localhost and 3000.
So far, so good and everything works fine but when email invitations are tried, it carries the .LinkUrl as http://localhost:3000 which is not exactly the url one would like to see.
If letting the email customization is cumbersome, please externalize the .LinkUrl attribute.

**Comment by peterholmberg** [comment]
While we are prioritizing this could this be of any help? https://community.grafana.com/t/how-to-edit-the-default-invitation-email-template/11940/2

**Comment by vlaguduva** [comment]
Thanks for the link; I have already looked at that post but changing the file content is not feasible as I said, we leave the grafana setup as it is except the nginx/ingress part.

**Comment by jorgefrater** [comment]
if you change the "domain" value under "server" in "grafana.ini" that will become the host for the URLs in the emails.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by saketbairoliya2** [comment]
Hi all, did we made any further progress on this?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
